Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -453,7 +525,13 @@ Extra: newpass=12345 (java.lang.String)
 
 #### Remediation
 
--- TODO [Describe the best practices that developers should follow to prevent this issue.] --
+If not strictly required, be sure that your IPC does not have the `android:exported="true"` value in the `AndroidManifest.xml` file nor an `<intent-filter>`, as otherwise this allows all other apps on Android to communicate and invoke it.
```

Comment:
I think this sentence is a bit confusing.

> your IPC does not have the ... value

Does IPC mean inter-process communication here? The communication itself doesn't really have a value. Maybe you meant the IPC configuration?

> as otherwise this allows

Maybe you can replace this with "to prevent".

> apps to communicate and invoke it.

I am not clear on what "it" is in this part of the sentence. What does the communicate mean here?